README.md

```diff
@@ -40,7 +40,7 @@ This plugin supports the following popular secrets backends:
 
 ## Installation
 
-> Nautobot Secrets Providers is compatible with Nautobot 1.2.0 and higher. Support for Thycotic Secret Server requires Python 3.7 or later.
+> Nautobot Secrets Providers is compatible with Nautobot 1.4.0 and higher. Support for Thycotic Secret Server requires Python 3.7 or later.
```

Comment:
Same comment as above

Reply:
Leaving as is

Reply:
We can link compatibility matrix instead of specific version numbers in the README WDYT?